Formula/certbot.rb

```diff
@@ -122,28 +144,18 @@ class Certbot < Formula
   end
 
   resource "six" do
-    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
-    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
   end
 
   resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/d7/8d/7ee68c6b48e1ec8d41198f694ecdc15f7596356f2ff8e6b1420300cf5db3/urllib3-1.26.3.tar.gz"
-    sha256 "de3eedaad74a2683334e282005cd8d7f22f4d55fa690a2a1020a416cb0a47e73"
+    url "https://files.pythonhosted.org/packages/94/40/c396b5b212533716949a4d295f91a4c100d51ba95ea9e2d96b6b0517e5a5/urllib3-1.26.5.tar.gz"
+    sha256 "a7acd0977125325f516bda9735fa7142b909a8d01e8b2e4c8108d0984e6e0098"
   end
 
   resource "zope.component" do
-    url "https://files.pythonhosted.org/packages/5d/e7/14935f56295e6a9a4ebcc82e30403f4bf580760801ea7ac6bae61ef0267b/zope.component-4.6.2.tar.gz"
-    sha256 "91628918218b3e6f6323de2a7b845e09ddc5cae131c034896c051b084bba3c92"
-  end
-
-  resource "zope.deferredimport" do
-    url "https://files.pythonhosted.org/packages/b9/74/6eb2dcf013fac35d086abef2435b5a6621435c2b0c166ef5b63a1b51e91d/zope.deferredimport-4.3.1.tar.gz"
-    sha256 "57b2345e7b5eef47efcd4f634ff16c93e4265de3dcf325afc7315ade48d909e1"
-  end
-
-  resource "zope.deprecation" do
-    url "https://files.pythonhosted.org/packages/34/da/46e92d32d545dd067b9436279d84c339e8b16de2ca393d7b892bc1e1e9fd/zope.deprecation-4.4.0.tar.gz"
```

Comment:
May need someone to double-check if these are safe to remove.

I originally ran `brew update-python-resources` for certbot.
And then had to re-added dependencies for `certbot-nginx/certbot-apache`:
- `python-augeas`
- `pyparsing`

Couldn't find:
- `zope.deferredimport`
- `zope.deprecation`
- `zope.proxy`

Reply:
Can the additionational resources be added to `pypi_formula_mappings.json`?

Reply:
Not sure if GitHub download tar will work well with pypi update.

One option I am considering:
- Use pypi URL for `certbot`
- Add `certbot-apache` and `certbot-nginx` as extra packages
- Install into virtualenv

The main pros is it is easier to bump with automatic resource updates

A couple cons with this:
1. It doesn't allow locking version of extra packages (e.g. `certbot-apache` unversioned as latest); though, in theory they should probably be updated at same time
2. It doesn't guarantee deterministic behavior as `certbot-apache`/`certbot-nginx` dependencies are not fixed as resources and instead installed via `pip`'s dependency logic.